GenderPayGap.WebUI/Search/SearchRepository.cs

```diff
@@ -19,6 +19,8 @@ public class SearchRepository
         private static List<SearchCachedUser> cachedUsers;
         private static DateTime cacheLastUpdated = DateTime.MinValue;
 
+
+        #region Load data into cache

```

Comment:
I'm still unsure about the use of `#regions`. It is a code smell and one that I was hoping to slowly phase out of the codebase. But I can see that it does a good job here of grouping things together and the file size is still relatively small.

I guess this is just a low-level grumble...